.bazelrc

```diff
@@ -43,12 +43,11 @@ coverage:coverage_clang --repo_env=BAZEL_LLVM_COV=/usr/bin/llvm-cov
 # cached previously. See https://bazel.build/configure/coverage?hl=en#remote-execution
 coverage:coverage_clang --nocache_test_results
 
-# Verilator is built for 4 cores and can time out if bazel is running more
-# than 1 test per four cores.
-# Until we get the built-in tag "cpu:4" to work for verilator tests, we can run
-# 1 test per 4 cores to avoid verilator performance falloff and timeouts we'd
-# otherwise see on machines with fewer than 40 cores.
-test --local_test_jobs=HOST_CPUS*0.25
+# # Uncomment this or add to ~/.bazelrc to override resource constrained test
```

Comment:
Nit: I personally find the doubly-commented lines a little distracting. I think it would be just as clear with singly-commented lines, and possibly newlines before and after the `test --local_test_jobs...` line.

Reply:
I've made it a config flag so it's clear how to use it and doesn't run by default.

Reply:
Even better!